resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -36,12 +36,16 @@ templates:
     target: ext/{{{project}}}.logrotate-legacy.conf
   - source: ext/ezbake.tmpfiles.conf.erb
     target: ext/{{{project}}}.tmpfiles.conf
+  - ext/fpm.rb.erb
+  - ext/redhat/preinst.erb
+  - ext/redhat/postinst.erb
+  - ext/redhat/prerm.erb
+  - ext/redhat/postrm.erb
```

Comment:
For RPM they live in the spec file, but FPM takes them as external files then uses those to generate the specs